types/react-native/index.d.ts

```diff
@@ -5035,6 +5036,11 @@ export class TouchableNativeFeedback extends TouchableNativeFeedbackBase {
     static canUseNativeForeground(): boolean;
 }
 
+export class TVEventHandler {
+  enable(context: any, callback: Function): undefined;
```

Comment:
is there a specific reason to use `undefined` instead of `void`?
```suggestion
  enable(context: any, callback: Function): void;
```